hexrd/ui/main_window.py

```diff
@@ -1025,23 +1029,33 @@ def enforce_view_mode(self, raw_only):
         if raw_only:
             self.image_mode_widget.ui.tab_widget.setCurrentIndex(0)
 
-    def apply_lorentz_polarization_correction_toggled(self, b):
+    def apply_polarization_correction_toggled(self, b):
         if not b:
             # Just turn it off and return
-            HexrdConfig().apply_lorentz_polarization_correction = b
+            HexrdConfig().apply_polarization_correction = b
             return
 
         # Get the user to first select the Lorentz polarization options
-        d = LorentzPolarizationOptionsDialog(self.ui)
+        d = PolarizationOptionsDialog(self.ui)
         if not d.exec_():
             # Canceled... uncheck the action.
-            action = self.ui.action_apply_lorentz_polarization_correction
+            action = self.ui.action_apply_polarization_correction
             action.setChecked(False)
             return
 
+        # The dialog should have modified HexrdConfig's polarization
+        # options already. Just apply it now.
+        HexrdConfig().apply_polarization_correction = b
+
+    def apply_lorentz_correction_toggled(self, b):
+        if not b:
+            # Just turn it off and return
+            HexrdConfig().apply_lorentz_correction = b
+            return
+
         # The dialog should have modified HexrdConfig's Lorentz options
         # already. Just apply it now.
-        HexrdConfig().apply_lorentz_polarization_correction = b
+        HexrdConfig().apply_lorentz_correction = b
```

Comment:
I think this whole function can just be `HexrdConfig().apply_lorentz_correction = b`

Reply:
Aah yes, I'll push these changes.